packages/db/src/search/CollectionSearch.ts

```diff
@@ -34,11 +50,19 @@ export class CollectionSearch extends BaseSearch {
       countQuery: Knex.QueryBuilder,
       searchQuery: Knex.QueryBuilder,
     } {
+    const {
+      granules: granulesTable,
+    } = TableNames;
     const countQuery = knex(this.tableName)
       .count(`${this.tableName}.cumulus_id`);
 
     const searchQuery = knex(this.tableName)
       .select(`${this.tableName}.*`);
+
+    if (this.searchIncludeStats()) {
+      searchQuery.join(`${granulesTable}`, `${this.tableName}.cumulus_id`, `${granulesTable}.collection_cumulus_id`)
+        .select(`${this.tableName}.*`, knex.raw(`ARRAY_AGG(${granulesTable}.status) AS statuses`)).groupBy(`${this.tableName}.cumulus_id`);
+    }
```

Comment:
what's the corresponding SQL?
Is `${this.tableName}.*` selected twice?
The endpoint returns all collections with or with granules, so it should be `leftJoin`

Reply:
ARRAY_AGG will create a huge list if there are many granules, we should use `count`